examples/components/hashicorp_vault.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: vault
+spec:
+  type: secretstores.hashicorp.vault
+  metadata:
+  - name: vaultAddr
+    value: "http://127.0.0.1:8200"
+  - name: skipVerify
+    value : true
+  - name: vaultTokenMountPath
+    value : ".hashicorp_vault_token"
```

Comment:
It expects this file to present at the same folder that the user runs `dapr run`, which is the file created above. Running this sample from this folder is documented as a required step (same as we have for bindings to have kafka setup).